src/Chapter2.hs

```diff
@@ -740,7 +760,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]] 
+contains n l = filter (elem n) l
```

Comment:
That's correct 💯 

You can see, that eta-reduction on `l` is possible here as well 🙂